CHANGES.rst

```diff
@@ -12,7 +12,7 @@
 Change History
 ##############
 
-:v2021.10:  expected *2021-10*
+:v2022.03:  expected *2022-03*
 
     * `#955 <https://github.com/nexusformat/definitions/issues/955>`_
        Documentation built and published from GitHub Actions workflow
```

Comment:
Next item in this list uses wrong issue - it should be #952

Reply:
I'll change the text.  It's a typo.  Note the branch is named for #952.